conflicts/Cargo.lock

```diff
@@ -771,7 +818,7 @@ dependencies = [
 [[package]]
 name = "octocrab"
 version = "0.17.0"
-source = "git+https://github.com/XAMPPRocky/octocrab?branch=master#9a677bf39167477e0b9dfa298dae03b3d6c19160"
+source = "git+https://github.com/XAMPPRocky/octocrab?branch=master#cc93dd39f0085154be4750863f8197d7637f644e"
```

Comment:
no need to bump here?

Reply:
I just updated everything, is this an issue?
The conflicts crate uses the `util` crate so wouldn't it create conflicting crates?

Reply:
Every crate compiles all dependencies from its own lockfile for itself. So different lockfiles for different crates shouldn't be an issue, as long as the dependencies inside the crate don't conflict with each other.

Reply:
Fixed, thanks.